packages/framework/aqueduct/src/container-runtime-factories/baseContainerRuntimeFactory.ts

```diff
@@ -73,13 +72,8 @@ export class BaseContainerRuntimeFactory
 		existing: boolean,
 	): Promise<ContainerRuntime> {
 		const scope: Partial<IProvideFluidDependencySynthesizer> = context.scope;
-		const dc = new DependencyContainer<FluidObject<IContainerRuntime>>(
-			this.dependencyContainer,
-			scope.IFluidDependencySynthesizer,
-		);
-		scope.IFluidDependencySynthesizer = dc;
-
-		const runtime: ContainerRuntime = await ContainerRuntime.loadRuntime({
+		scope.IFluidDependencySynthesizer = this.dependencyContainer;
```

Comment:
this doesn't look right. if they both exists (this.dependencyContainer, scope.IFluidDependencySynthesizer) we need to merge them. if only one exists, you can only pass it along.

Reply:
I looked through ff and office-bohemia codebases. There are no cases where we pass a value other than undefined to the constructor for dependencyContainer. I have thus removed it.
I should probably remove it from ContainerRuntimeFactoryWithDefaultDataStore as well but that class is used all over tests. Once we have consensus on this approach I will go ahead and do the associated changes.